sdk/rust/oak/src/grpc/mod.rs

```diff
@@ -236,7 +236,7 @@ where
 {
     // Create a new channel for request message delivery.
     let (req_sender, req_receiver) =
-        crate::io::channel_create::<GrpcRequest>(&Label::public_untrusted())
+        crate::io::channel_create::<GrpcRequest>("gRPC request", &Label::public_untrusted())
```

Comment:
What is the difference between this and 
```
 .channel_create("gRPC request", &label)
```
in `oak_runtime/src/node/grpc/server/mod.rs`? Does one of these need a different name?

Reply:
I believe this is the equivalent that can used by a Wasm node to send gRPC invocations out to the gRPC client pseudo node. I think that it is fine for both to have the same name as they would be used in different contexts, which should make clear whether it is a gRPC client or server pseudo node.